server/src/test/java/io/crate/integrationtests/FixCorruptedMetadataCommandITest.java

```diff
@@ -134,6 +136,7 @@ public void test_swap_table_partitioned_dotted_src_to_partitioned_dotted_target(
     }
 
     @Test
+    @Repeat (iterations = 20)
```

Comment:
Please don't forget to remove, also on the test below.

Reply:
right, forgot, thx!